docs/eslint/eslint-plugin-query.md

```diff
@@ -23,7 +23,7 @@ Add `@tanstack/eslint-plugin-query` to the plugins section of your `.eslintrc` c
 
 ```json
 {
-  "plugins": ["@tanstack/query"]
+  "plugins": ["@tanstack/eslint-plugin-query"]
```

Comment:
I think this is correct as it is. You can leave out the `eslint-plugin` prefix when adding plugins 🤷 

@Newbie012 ?

Reply:
Correct. I believe there are examples that are using the plugin according to the current docs.

Reply:
Sorry, you're right, I'm confusing plugins with extends 😅